cherrypy/test/test_caching.py

```diff
@@ -272,6 +286,40 @@ def testExpiresTool(self):
         if cherrypy.server.protocol_version == 'HTTP/1.1':
             self.assertHeader('Cache-Control', 'no-cache, must-revalidate')
         self.assertHeader('Expires', 'Sun, 28 Jan 2007 00:00:00 GMT')
+        
+    def testGzipStaticCache(self):
+        headers = [('Accept-Encoding', 'gzip')]
+        idx_uri = '/gzip_static_cache/index.html'
+        jpg_uri = '/gzip_static_cache/dirback.jpg'
+        
+        self.getPage(idx_uri, method='GET', headers=headers)
+        idx_resp_headers = dict(self.headers)
+        idx_gz_content_len = idx_resp_headers['Content-Length']
+        
+        self.getPage(jpg_uri, method='GET', headers=headers)
+        jpg_resp_headers = dict(self.headers)
+        jpg_gz_content_len = jpg_resp_headers['Content-Length']
+        
+        for _ in range(3):
```

Comment:
Is if required to have 3 queries? Wouldn't it be enough to have 2?

Reply:
Ah yes, you're right. Updated in latest commit.